### PR TITLE
Add fallback for MultiPolygon#contains? on point

### PR DESCRIPTION
**Pull request by Quiwin**
### Summary

Add a fallback on `MultiPolygon#contains?`.

Since we have a fallback on `Polygon#contains?`, we might as well use it for a simple `MultiPolygon#contains?` on a point.

### CODE REVIEW

File: lib/rgeo/impl_helper/basic_geometry_collection_methods.rb [hunk 1]
```diff
@@ -177,6 +177,19 @@ def boundary
       def coordinates
         @elements.map(&:coordinates)
       end
+
+      def contains?(rhs)
+        unless Feature::Point === rhs
+          raise(Error::UnsupportedOperation,
+            "Method MultiPolygon#contains? is only defined for Point"
+          )
```
Comment (BuonOmo) [inline lib/rgeo/impl_helper/basic_geometry_collection_methods.rb:185]:
```suggestion
          super
```

There may be a super method handling that

File: lib/rgeo/impl_helper/basic_geometry_collection_methods.rb [hunk 2]
```diff
@@ -177,6 +177,19 @@ def boundary
...
+        end
+
+        @elements.each do |poly|
+          return true if poly.contains?(rhs)
+        end
+        false
```
Comment (BuonOmo) [inline lib/rgeo/impl_helper/basic_geometry_collection_methods.rb:191]:
```suggestion
        @elements.any? { |polygon| polygon.contains?(rhs) }
```

Some rubyification/simplification :)

**Review by BuonOmo — APPROVED** [review]
LGTM! @keithdoggett I'll let your review and merge if you are also ok with the feature :)

@quiwin I've proposed 2 changes, could you review those ?

**Comment by keithdoggett** [comment]
Looks good! Thanks for submitting this. What are your thoughts on extending this to include `MultiLineStrings` as well? In your last PR (#224), you added `LineString#contains?` so it should be simple enough to make it work for `MultiLineStrings`.

**Review by keithdoggett — APPROVED** [review]

**Pull request merged**